build.gradle

```diff
@@ -254,7 +258,66 @@ project(':samza-elasticsearch') {
     testCompile "junit:junit:$junitVersion"
     testCompile "org.mockito:mockito-all:$mockitoVersion"
 
-    // Logging in tests is good.
+    testRuntime "org.slf4j:slf4j-simple:$slf4jVersion"
+  }
+}
+
+project(':samza-sql-api') {
```

Comment:
I strongly believe cleaner separation between APIs and implementation. Since UDFs are loaded at runtime, if users creating udfs bring in implementation code along with their UDFs there would be a version mismatch hell between the UDFs and the core impl code when we load the UDFs.

We could consider merging samza-sql-lib and Samza-sql, But this will make our msg convertor plugins only compile time plugins and any version mismatch should be handled at compile time. IMO samza-sql-udf should be as thin as possible, Does this make sense?